src/main/java/com/server/ResponseBodyBuilder.java

```diff
@@ -1,75 +1,80 @@
 package com.server;
 
+import javax.imageio.ImageIO;
+import javax.imageio.stream.ImageInputStream;
 import java.io.*;
 
 public class ResponseBodyBuilder {
-    private String body = "";
-    private String publicDir = "";
+    private byte[] body;
     private RequestRouter requestRouter;
     public ResponseParams responseParams;
 
     ResponseBodyBuilder(RequestRouter requestRouter){
         this.requestRouter = requestRouter;
     }
 
-    public String getBody(RequestParams requestParams, ResponseParams responseParams) throws IOException {
+    public byte[] getBody(RequestParams requestParams, ResponseParams responseParams) throws IOException {
         this.responseParams = responseParams;
-
+        String path = requestParams.getPath();
         if (requestParams.getPath().equals("/file1")) {
```

Comment:
I think this one can be changed also from requestParams.getPath() to just path.  Also was thinking this method could be slightly better encapsulated to standardize like the others.  But I can take care of both these items in my next PR.